copilot/fastapi/tests/test_tvp.py

```
@@ -0,0 +1,141 @@
+import pytest
+from unittest.mock import patch, MagicMock
+from pathlib import Path
+import os
+import threading
+
+def test_tvp_status_endpoint(test_client, mock_settings):
+    """Test the TVP status endpoint"""
+    # Mock Path.exists and Path.iterdir
+    with patch("pathlib.Path.exists") as mock_exists, \
+         patch("pathlib.Path.iterdir") as mock_iterdir, \
+         patch("pathlib.Path.is_dir") as mock_is_dir, \
```

Ruff (0.8.2)

Local variable `mock_is_dir` is assigned to but never used

Remove assignment to unused variable `mock_is_dir`

(F841)